app/script/conversation/ConversationRepository.coffee

```diff
@@ -621,23 +621,30 @@ class z.conversation.ConversationRepository
         callback next_conversation_et
       else
         @archive_conversation conversation_et, next_conversation_et
-    .catch (error) =>
-      @logger.error "Failed to leave conversation (#{conversation_et.id}): #{error}"
 
   ###
-  Remove member from conversation.
+  Remove bot from conversation.
+  @param conversation_et [z.entity.Conversation] Conversation to remove member from
+  @param user_id [String] ID of bot to be removed from the the conversation
+  ###
+  remove_bot: (conversation_et, user_id) =>
+    @conversation_service.delete_bots conversation_et.id, user_id
+    .then (response) ->
+      if response
```

Comment:
Currently there is not response ever from the backend. I requested to get the expected `conversation.member-leave` event in the 200 reponse just like we do on the delete member endpoint.